src/functions.php

```diff
@@ -20,13 +20,31 @@
  *
  * @param class-string<TObject> $class
  *
+ * @deprecated
+ *
  * @return AnonymousFactory<TObject>
  */
 function factory(string $class, array|callable $defaultAttributes = []): AnonymousFactory
 {
+    trigger_deprecation('zenstruck\foundry', '1.30', 'Usage of "factory()" function is deprecated and will be removed in 2.0. Use functions "create()", "instantiate()" or "repository()" instead.');
+
     return new AnonymousFactory($class, $defaultAttributes);
 }
 
+/**
+ * @see Factory::__construct()
+ *
+ * @template TObject of object
+ *
+ * @param class-string<TObject> $class
+ *
+ * @return Factory<TObject>
+ */
+function anon(string $class, array|callable $defaultAttributes = []): Factory
```

Comment:
not 100% sure about this method name :shrug:

Reply:
Maybe `anonymous` to be explicit? I'd be cool to rename this back to factory at some point post 2.0.